src/TestEnvironment.Docker.Containers.Elasticsearch/TestEnvironment.Docker.Containers.Elasticsearch.csproj

```diff
@@ -5,7 +5,7 @@
     <LangVersion>Latest</LangVersion>
     <AssemblyName>TestEnvironment.Docker.Containers.Elasticsearch</AssemblyName>
     <RootNamespace>TestEnvironment.Docker.Containers.Elasticsearch</RootNamespace>
-    <Version>1.0.3</Version>
+    <Version>1.0.4</Version>
```

Comment:
Why do you need these packages updates?

Reply:
If you reference for example `TestEnvironment.Docker.Containers.Elasticsearch` you'll get a reference to `TestEnvironment.Docker` recursively. The problem is that we upgraded `TestEnvironment.Docker` but didn't touch other packages and right now they have references to previous version of it. It can be solved by direct reference to `TestEnvironment.Docker` but it is definitely undesirable.